src/scene_battle_rpg2k3.cpp

```diff
@@ -1277,6 +1284,30 @@ void Scene_Battle_Rpg2k3::SpecialSelected() {
 	ActionSelectedCallback(active_actor);
 }
 
+void Scene_Battle_Rpg2k3::RowSelected() {
+	// Switching rows is only possible if in back row or
+	// if at least 2 party members are in front row
+	int current_row = active_actor->GetBattleRow();
+	int front_row_battlers = 0;
+	if (current_row == Game_Actor::RowType::RowType_front) {
+		for (auto& actor: Main_Data::game_party->GetActors()) {
+			if (actor->GetBattleRow() == Game_Actor::RowType::RowType_front) front_row_battlers++;
+		}
+	}
```

Comment:
This case doesn't appear to handle back attacks etc..

I checked in RE tools and found that RPG_RT checks `row == mirrored` to mean front `row != mirrored` where you say back.

Reply:
Did the change as requested and tested it. I have amended the commit and force pushed it because I assume you want this in one commit.